### PR TITLE
View options

### PR DESCRIPTION
**Pull request by jesse-wei**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -25,6 +25,6 @@
     "MIN_HEIGHT_RATIO": 0.5,
     // Maximum width and height of the GUI as a ratio of primary monitor size.
     // Enter a number between 0 (not inclusive) and 1.
-    "MAX_WIDTH_RATIO": 0.6,
-    "MAX_HEIGHT_RATIO": 0.6
+    "MAX_WIDTH_RATIO": 0.9,
+    "MAX_HEIGHT_RATIO": 0.9
```
[the remark this answers is not in the document]
Comment (jesse-wei) [reply config.json:29]: It be like that. Was hoping you Windows users wouldn't notice :P

Does that make it look bad on your computer though?

**Review by madiali — COMMENTED** [review]
There is a minor bug when loading multiple images.

To replicate:
1. Add multiple images with view set to "Z view"
2. Change to "X view" or "Y view" on 1st image
3. Go to next image

**Comment by jesse-wei** [comment]
> There is a minor bug when loading multiple images.
> 
> To replicate:
> 
> 1. Add multiple images with view set to "Z view"
> 2. Change to "X view" or "Y view" on 1st image
> 3. Go to next image

Great catch!

It's because next and previous don't call orient_image (which is currently called only in update_view). Pressing X or Y view again orients it correctly. I'll fix the code logic here.

Also, do we want to completely remove SMOOTH_BEFORE_RENDERING? There are a few more instances.

**Comment by madiali** [comment]
Yes, remove SMOOTH_BEFORE_RENDERING

**Comment by jesse-wei** [comment]
Good to merge?

**Pull request merged**